game.js

```diff
@@ -369,6 +371,11 @@ class Game {
 }
 
 function main() {
+
+  const WIDTH = 25;
```

Comment:
constants should live on the module level, not a local variable - otherwise they would just be a local variable